content/docs/guides/continuous-delivery/octopus-deploy.md

```diff
@@ -0,0 +1,110 @@
+
+## Stack and Branch Mappings
+
+The steps below act on a hypothetical stack: `my-org/my-project/aws-ts-hello-fargate`.
+You can create a new stack by running [`pulumi stack init`]({{< relref "/docs/reference/cli/pulumi_stack_init.md" >}}) from the folder containing the `Pulumi.yaml` file.
+
+**Note**: The names used above are purely for demonstration purposes only.
+You may choose a naming convention that best suits your organization.
+
+## Octopus Setup
+
+### Infrastructure
+
+Infrastructure in Octopus is represented as environments, deployment targets and workers (tentacles or SSH machines.) You could think of each environment as representing one of your Pulumi stacks. For example, if you are creating cloud infrastructure that represents your dev, staging and prod environments, each of those would typically map to a [Pulumi stack](https://www.pulumi.com/docs/intro/concepts/stack/) and you could create an Octopus environment for each of those.
+
+In a typical scenario where Pulumi is creating your cloud infrastructure, you will only need to a worker that can run the Pulumi CLI commands against your code package.
+
+### Project
+
+A project represents the application being deployed and uses versioned packages (detailed below) as well as variables, during the deployment process. A project also contains a deployment process, which is a series of steps that are associated with "deploying" your project. Projects can contain the actual runtime application (a Go service, or a Python Flask app, etc.), as well as any infrastructure required.
+
+### Packages
+
+Packages can be your source code bundled-up in one of the [supported formats](https://octopus.com/docs/packaging-applications#supported-formats). They can also just be a handlful of scripts that you may want to use during your deployment process.
+
+In order to create a package, Octopus offers several ways that you can integrate into your existing build (CI) system. [Learn more](https://octopus.com/docs/packaging-applications/create-packages) about the options available to you for packaging your apps.
+
+For Pulumi apps, you can simply package the entire Pulumi app and extract the bundled package onto a worker where the Pulumi CLI can access them.
+
+## Deployment Process
+
+In order to create an [Octopus deployment process](https://octopus.com/docs/deployment-process), project variables need to be configured for each of the environments which maps to a Pulumi stack. Each Pulumi stack could use environment-specific configuration. For example, each stack (or environment) could be deployed using different cloud credentials. So at a minimum you would need to configure those credentials needed for deploying a Pulumi stack.
```

Comment:
Looks like I realized mid-way through the sentence that I need to write about how about Octopus environments map to Pulumi stacks and didn't come back to finish what I wanted to write here. Finished my thought now, though :)